Entity/SeasonRepository.php

```diff
@@ -45,4 +45,23 @@ public function findByPerimeter(Perimeter $perimeter)
 
         return $qb->getQuery()->getResult();
     }
+
+    /**
+     * Find Season(s) by title
+     *
+     * @param String $title
+     *
+     * @return Doctrine\Common\Collections\Collection Collection of seasons
+     */
+
+    public function findByTitle($title)
```

Comment:
this method is useless 
see https://github.com/CanalTP/MttBundle/pull/75/files#diff-46a67f301fd2289e0ca6e4b816a533e2R156